src/screens/DiscoveryScreenV2/index.js

```diff
@@ -311,13 +311,13 @@ const DiscoveryScreenV2 = ({route}) => {
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
     flex: 1,
     paddingTop: 60
   },
   fragmentContainer: {
     height: '100%',
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on replacing the hard-coded color values with constants from a theme file. This will make it easier to maintain and update the color scheme in the future.
```diff
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```
```diff
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->